metric-publishers/emf-metric-logging-publisher/src/main/java/software/amazon/awssdk/metrics/publishers/emf/internal/EmfMetricConfiguration.java

```diff
@@ -45,10 +47,12 @@ public final class EmfMetricConfiguration {
 
     private EmfMetricConfiguration(Builder builder) {
         this.namespace = builder.namespace == null ? DEFAULT_NAMESPACE : builder.namespace;
-        this.logGroupName = builder.logGroupName;
+        this.logGroupName = resolveLogGroupName(builder);
```

Comment:
nit: `this.logGroupName = Validate.paramNotNull(resolveLogGroupName(builder), "logGroupName"`